x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/plugin/EqlPlugin.java

```diff
@@ -46,6 +46,8 @@
 import java.util.function.Supplier;
 
 public class EqlPlugin extends Plugin implements ActionPlugin {
+    // We are going to reuse the same index as normal async search until system indices are implemented
+    public static final String INDEX = ".async-search";
```

Comment:
We should share this in core next to AsyncTaskMaintenanceService ?
I also wonder who should be responsible to start the `AsyncTaskMaintenanceService` since we don't need to start it twice ?

Reply:
Yes. Good point. Let's deal with it in the next PR when I start moving stuff around.